src/genesis.rs

```diff
@@ -96,4 +197,119 @@ impl Genesis {
         }
         Ok(())
     }
+
+    fn setup_genesis_flags(&self) -> Vec<String> {
+        let mut args = vec![
+            "--bootstrap-validator-lamports".to_string(),
+            sol_to_lamports(
+                self.flags
+                    .bootstrap_validator_sol
+                    .unwrap_or(DEFAULT_BOOTSTRAP_NODE_SOL),
+            )
+            .to_string(),
+            "--bootstrap-validator-stake-lamports".to_string(),
+            sol_to_lamports(
+                self.flags
+                    .bootstrap_validator_stake_sol
+                    .unwrap_or(DEFAULT_BOOTSTRAP_NODE_STAKE_SOL),
+            )
+            .to_string(),
+            "--hashes-per-tick".to_string(),
+            self.flags.hashes_per_tick.clone(),
+            "--max-genesis-archive-unpacked-size".to_string(),
+            self.flags
+                .max_genesis_archive_unpacked_size
+                .unwrap_or(DEFAULT_MAX_GENESIS_ARCHIVE_UNPACKED_SIZE)
+                .to_string(),
+            "--faucet-lamports".to_string(),
+            self.flags
+                .faucet_lamports
+                .unwrap_or(DEFAULT_FAUCET_LAMPORTS)
+                .to_string(),
+            "--faucet-pubkey".to_string(),
+            self.config_dir
+                .join("faucet.json")
+                .to_string_lossy()
```

Comment:
Sorry if this is dumb, but why are you using the `_lossy` variants in a few places?

Reply:
not all paths are UTF-8 encoded. See: https://stackoverflow.com/questions/37388107/how-to-convert-the-pathbuf-to-string. So I could avoid using `to_string_lossy()` and do something like the following instead: 
```
self.config_dir.join("faucet.json").into_os_string().into_string().unwrap()
```

Reply:
or i could do: `self.config_dir.join("faucet.json").into_os_string().into_string()?` and return a `Result`

Reply:
ah ok, thanks for the explanation! Totally ok to use the `_lossy` bits for now then, I didn't realize windows doesn't use legal utf-8 encoding for paths, that's wacky